server/plugin/command.go

```diff
@@ -358,7 +358,7 @@ func (p *Plugin) handleSubscribesAdd(_ *plugin.Context, args *model.CommandArgs,
 	}
 	repoLink := p.getBaseURL() + owner + "/" + repo
 
-	msg := fmt.Sprintf("Successfully subscribed to [%s](%s).", repo, repoLink)
+	msg := fmt.Sprintf("Successfully subscribed to [%s](%s) with events: %s.", repo, repoLink, features)
```

Comment:
I'm thinking we should announce which user created this subscription, like we do in the Jira plugin https://github.com/mattermost/mattermost-plugin-jira/blob/268b3e4f18b67fbe5f9c1d85b2054d50fba3ecd0/server/subscribe.go#L791

Reply:
![image](https://user-images.githubusercontent.com/86650698/144477459-37e04d6e-c8eb-4390-a2bb-b0961d7529c2.png)
@mickmister tried to replicate the message , does it looks good ?

Reply:
There should be spaces between the feature. 

Also: 0/5 Should they each be in a code block e.g. `Feature1`, `Feature2`,...

Reply:
@hanzei have a look on this 

![image](https://user-images.githubusercontent.com/86650698/144594836-e11819bb-7774-4ea3-93a8-f983d6921332.png)